### PR TITLE
Task 3 and 4

### PR DESCRIPTION
**Pull request by qnislavnachev**
About Task 3:  
Read file, put the information in buffer, reverse the info and save it in same file!
About Task 4:
Read file, copy the chosen bytes in a new file, user have to choose the length and offset.

### CODE REVIEW

File: src/task4/TransferObject.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class TransferObject {
+    public void transfer(InputStream in, OutputStream out, int from, int to) throws IOException {
+        byte[] bytes = new byte[1024];
+        in.read(bytes);
+        for (int i = from; i <= to; i++) {
+            System.out.println(bytes[i]);
+            out.write(bytes[i]);
+        }
+        in.close();
+        out.close();
```
Comment (EmilGeorgiev) [inline src/task4/TransferObject.java]: This which create InputStream and OutputStream it should close it

File: src/task3/Demo.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package task3;
+
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Demo {
+    public static void main(String[] args) throws IOException {
```
Comment (EmilGeorgiev) [inline src/task3/Demo.java]: your application should catch exception not throw from main method

**Comment by EmilGeorgiev** [comment]
LGTM after fix comment.

**Comment by qnislavnachev** [comment]
Sorry for that commit I wrong something! I will add new commit with full fixes!

**Pull request closed without merging**